core/src/test/kotlin/io/sharing/server/core/product/domain/ProductTest.kt

```diff
@@ -30,5 +33,26 @@ class ProductTest {
         assertThat(product.status).isEqualTo(ProductStatus.REGISTERED)
         assertThat(product.region).isEqualTo(region)
         assertThat(product.description).isEqualTo(description)
+        assertThat(product.images.size).isEqualTo(1)
+    }
+
+    @Test
+    fun `상품 이미지 추가`() {
+        val product = createProduct()
+
+        IntStream.rangeClosed(1, 10).boxed().forEach {
+            product.addImage(createProductImage("test-url$it"))
+        }
```

Comment:
```suggestion
        (1..10).forEach {
            product.addImage(createProductImage("test-url$it"))
        }
```

위 코드와 같이 작성하는 편이 더욱 코틀린스러울거같습니다.